### DISCUSSION TITLE
Buypass Go SSL - free TLS/SSL certificate

### DISCUSSION THREAD

**Discussion by fa-ve** [ZoeyVid/NPMplus#492] [Q&A] [answered]
Is there a possibility to use: [](https://www.buypass.com/products/tls-ssl-certificates/go-ssl) instead of Let's Encrypt?

**Comment by Zoey2936** [comment, marked as answer]
you can upload custom certificates or edit the default acme server in /opt/npm/tls/certbot/config.ini
